Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.yml

```
@@ -21,6 +21,15 @@ configuration:
   name: proxy
   required: false
   type: 8
+- defaultvalue: Elasticsearch
+  additionalinfo: In some hosted ElasticSearch environments (AWS), the standard ElasticSearch client is not supported. If your environment is not supported, please select OpenSearch as the parameter.
```

```suggestion
  additionalinfo: In some hosted ElasticSearch environments (AWS), the standard ElasticSearch client is not supported. If your environment is not supported, please try using the OpenSearch client type.
```

Are we sure that it happens in AWS env?

---

Please also add the default value of 'Elasticsearch'.